src/domain/app/AppConfig.ts

```diff
@@ -94,6 +94,19 @@ class AppConfig {
   static get showSimilarEvents() {
     return Boolean(parseEnvValue(process.env.NEXT_PUBLIC_SHOW_SIMILAR_EVENTS));
   }
+
+  static get URLTransformMapping() {
+    return {
+      'https://api.hel.fi/linkedevents/v1/event': '/courses',
```

Comment:
these domains probably should be read from env configuration?

Reply:
yea and they should not be that hard coded at all. I mean, we cannot have hard coded url routes for articles categories for example. They are too dynamic. 

This is the reason why this PR is still in draft mode. We need more specific URL's from the Headless CMS. The context needs to be told in the URL.